src/caches/basic_method_caches.jl

```diff
@@ -110,8 +110,9 @@ function alg_cache(alg::RKMilCommute,prob,u,ΔW,ΔZ,p,rate_prototype,noise_rate_
   du1 = zero(rate_prototype); du2 = zero(rate_prototype)
   K = zero(rate_prototype); gtmp = zero(noise_rate_prototype);
   L = zero(noise_rate_prototype); tmp = zero(rate_prototype)
-  I = zeros(length(ΔW),length(ΔW));
-  Dg = zeros(length(ΔW),length(ΔW)); mil_correction = zero(rate_prototype)
+  J = false .* vec(ΔW) .* vec(ΔW)'
```

Comment:
If it's J then it will clash with the common name for the Jacobian. Make it WikJ?

Reply:
Okay!